### PR TITLE
Use StdDev from db instead of statistics.py

### PR DESCRIPTION
**Pull request by jseppi**
Starting in hopes of addressing #519 

To Dos:
- [x] cleanup/remove unused old stddev code
- [x] validate that it is faster that statistics.py
- ~~figure out how to validate StdDev aggregation~~

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -177,26 +177,31 @@ def get(self, request):
...
-                current_rates.append(rate[wage_field])
-
-        if current_rates:
-            std_dev = stdev(current_rates)
-        else:
-            std_dev = None
-
-        page_stats['first_standard_deviation'] = std_dev
+        page_stats['first_standard_deviation'] = \
+            quantize(contracts_all.aggregate(
```
Comment (jseppi) [inline api/views.py]: Actually we could batch all these aggregations into a single call, like 

```py
models.aggregate(Avg('price'), Max('price'), Min('price'))
```

**Comment by jseppi** [comment]
@toolness Does this alleviate the performance problems you saw due to `statistics.py`?

**Comment by jseppi** [comment]
For reference, this is the commit where the numpy-based standard deviation was first introduced: https://github.com/18F/calc/pull/178/commits/cb6ec4f4699519a928a95b881730398f27b50022 
Perhaps the original developer just didn't know there was an available StdDev aggregation? I couldn't find any other discussion for why numpy was used.

**Comment by toolness** [comment]
NOICE

> @toolness Does this alleviate the performance problems you saw due to `statistics.py`?

Good question!  Maybe I should add some kind of `manage.py performance_test` command to make it easy to figure out?  Alternatively I could just do it manually.

**Comment by jseppi** [comment]
I tried for a couple minutes to get the DjDT perf panel to work but didn't figure out within a few minutes, so I punted on it.

**Comment by jseppi** [comment]
In manual testing I noticed that the standard deviations calculated were very slightly different, but only after like 8 decimal places, which seems insignificant to me.

**Comment by toolness** [comment]
Nice!

To get the DjDT perf panel to work, you have to [define it in `settings.py`](https://django-debug-toolbar.readthedocs.io/en/stable/panels.html#profiling), which is a bit annoying.  I guess we could add it so it always shows up.

Anyways, then you have to go to a page that uses it directly, rather than one that uses it via an ajax request, which probably means that you need to visit `/api/rates/` in your browser (which causes django rest framework to run the query as part of the page request and render it as html, along with the debug toolbar).

**Comment by jseppi** [comment]
Actually, we have tests already in place for testing the standard deviation values (in `test_api.py`), and now the values are calculated by Django/Postgres, I feel less need to validate results against the previous custom code.

**Comment by jseppi** [comment]
Here are the DjDT Profiling results on `/api/rates` now:

screen shot 2016-08-12 at 4 55 36 pm

Must faster than what is shown in #519, assuming I ran the profiling correctly.

File: api/utils.py [hunk 2]
```diff
@@ -1,24 +1,4 @@
 
-import math
-import statistics
```
Comment (toolness) [inline api/utils.py]: Heh, took me a little while to figure out that [`statistics`](https://docs.python.org/3/library/statistics.html) is a standard Python module introduced in 3.4. First I thought it was a module we wrote, then I thought it was a third-party module. 😁

**Comment by toolness** [comment]
Wow, this is *much* faster! The qunit test suite takes a really long time on my local machine with all the data loaded, because the `hourglass.API: #get()` test takes forever, but this PR makes it almost instantaneous. Great work!

File: api/utils.py (hunk 2 above)
Comment (jseppi) [inline api/utils.py]: Yeah, it looks like a nice package, but is apparently slow (at least slower than numpy and native db ops, at least).

**Pull request merged**